packages/utils/array-utils/index.js

```diff
@@ -30,4 +30,20 @@ const nth = (index, data) => {
   return data[index]
 }
 
-module.exports = {toArray, head, flatten, nth}
+/* function flatten (array) {
```

Comment:
See the flatten in modeling, which is recursive. I think that’s a better implementation.

Reply:
Ah yes, btw I did not touch the array things in modeling because there are other people doing changes in modeling, and we can always make modeling import array utils.
Btw about recursive vs non recursive : perhaps we should have both: I remember there was a case where a single level of flattening was expected (I cannot recall the exact location, sorry :( )

We could provide a 'flattenDeep' or 'flattenNested' or a parameter ?